rockets/jsonrpc/receiver.cpp

```diff
@@ -196,12 +195,12 @@ void Receiver::bindAsync(const std::string& method,
     _impl->methods[method] = action;
 }
 
-std::string Receiver::process(const std::string& request)
+std::string Receiver::process(Request request)
```

Comment:
Be very careful about replacing a const & param by an object that is passed by value. Many copies can occur along the way, and the request string could potentially be large.